data/json/items/containers.json

```
@@ -738,6 +738,16 @@
     "symbol": ")",
     "color": "brown"
   },
+  {
+    "id": "box_compact_wood",
+    "type": "GENERIC",
+    "name": { "str": "compact wooden box", "str_pl": "compact wooden boxes" },
+    "copy-from": "ammunition_can_30",
```

```suggestion
    "copy-from": "ammunition_can_30",
    "material": [ "wood" ],
```
No steel & aluminum wooden boxes for you. 😛